### ISSUE TITLE
Values carried from Table 3 to 4 even when normalization is not possible.

### ISSUE THREAD

**Issue by ismailoglu** [ismailoglu/RT-analyzer#7]
Using Sample set 3, when Day 0 NT-1 sample is picked for normalization (no biological replicates), same globin values appear on both tables. (Relative Expression Value by Probe and Relative Expression Value by Probe Normalized by Control Sample or Control Biological Replicates Group) In the second table the values in Globin column should be N/A, because we can't normalize globin values based on the Day 0 NT-1 sample.

However, when Day 3 NT-1 sample is picked, OCT NANOG and BRACHYURY samples are shown as N/A on the second table (Relative Expression Value by Probe Normalized by Control Sample or Control Biological Replicates Group) as they should. When Day 0 NT-2 sample is picked, the app behaves correctly again. (N/As for NKX, FOXA2 and GLOBIN) 

So, the issue seems to be isolated to the Day 0 NT-1 sample. It might be due to the three empty GLOBIN wells that are ignored.

**Comment by slacktracer** [comment]
@ismailoglu Thank you!

I'll look into it ASAP (this coming week hopefully).

**Comment by slacktracer** [comment]
Hi, @ismailoglu 

Did we fix this? I think so but I can't remember. =P

Let me know if you know. I won't have time right now to check.

**Comment by ismailoglu** [comment]
This seems to be fixed. So, I am closing it. Thanks

**Comment by ismailoglu** [comment]
I thought this was fixed, but it actually was not. Can you look into it when you get a chance?

To be able to see the tables I am talking about you need to remove all the biological replicates groups after loading the example dataset from the bottom of the page.

Let me know if it is not clear and I can write it in more detail.

**Comment by slacktracer** [comment]
@ismailoglu 

Hey, followed your instructions carefully. I think I got it...

You said: _In the second table the values in Globin column should be N/A_

You mean, ALL values in the column, right?

I checked the other situations, using Day 3 NT-1 and Day 0 NT-2 and that's the difference. There all the values are N/A...

So the correct would be having all values in column Globin being N/A for Day 0 NT-1. Despite the fact that in the other columns some values are N/A and some are not?

I think I am understanding it... =)

I think I know what's going on, though not how to fix it yet. =P

Can you, please, confirm or deny?

**Comment by ismailoglu** [comment]
Yep, I think you got the issue. They should all be N/A once normalized to Globin from Day0 NT-1, because that value itself is N/A.

**Comment by slacktracer** [comment]
@ismailoglu 

Great! I will go into the code now. If don't come back in a few days send someone!

Just kidding, the code is not **that** messy. =P

**Comment by slacktracer** [comment]
@ismailoglu 

I think I got it. There was a problem with having the GLOBIN probe on the plate but all its values missing. It did not happen with NKX because only one value of three is missing...

(comments on the commit explain the problem a little: https://github.com/ismailoglu/RT-analyzer/commit/46f24f63794b555ad942bd6fb8fd26a328717bd8)

So, it seems to be working fine now. =)

Take a look when you can and let me know!

**Comment by ismailoglu** [comment]
Seems fixed to me! 
Thank you!
